backend/tests/test_rules_config_integration.py

```
@@ -532,3 +537,58 @@ def test_action_engine_honors_non_default_expiration_warning_days():
+    _seed_position(client, broker_cost_basis=3800.0)
+    _seed_setting(
+        client,
+        RULES_CONFIG_KEY,
+        _full_rules_config(
+            position={"sizing_cap_pct": 25.0, "sizing_cap_account": None}
+        ),
+    )
+    monkeypatch.setattr(
+        positions_router,
+        "get_cached_account_value",
+        lambda db, sizing_cap_account=None: AccountValueResult(
```

Ruff (0.15.13)

[warning] Unused lambda argument: `db`

(ARG005)